en/docs/develop/configs-rest-api.md

```diff
@@ -0,0 +1,39 @@
+---
+template: templates/swagger.html
+---
+# Server Configurations API definition - v1
+
+??? Note "Click For Instructions"
+    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
```

Comment:
Follow the instructions given below to try out the REST APIs with your local instance of WSO2 Identity Server (WSO2 IS).